docs/src/main/paradox/testing-testcontainers.md

```diff
@@ -0,0 +1,84 @@
+---
+project.description: Alpakka Kafka provides Testcontainers support for running a Kafka cluster locally using Docker containers.
+---
+# Testing with a Docker Kafka cluster
```

Comment:
This becomes the title in the ToC, as well. 
Maybe "Testing using Testcontainers".
Or have it as a sub-page to `testing.md` might be even better.

Reply:
I may have misunderstood what you meant about having it as a separate page.  I thought you wanted it to be highlighted in the ToC as well, otherwise why separate it?

Reply:
Resolved offline.